bin/fillTestData/Tribes.js

```diff
@@ -4,8 +4,14 @@ var path = require('path'),
     mongooseService = require(path.resolve('./config/lib/mongoose')),
     chalk = require('chalk'),
     faker = require('faker'),
+    fs = require('fs'),
+    tribesData = JSON.parse(fs.readFileSync(path.resolve('./bin/fillTestData/data/Tribes.json'), 'utf8')),
     mongoose = require('mongoose');
 
+var random = function (max) {
+  return Math.floor(Math.random() * max);
+};
```

Comment:
Since we have `lodash` at our hands,  `_.sample` could work smoothly here?

https://lodash.com/docs/4.17.10#sample